real-main/app/models/user/model.py

```diff
@@ -470,3 +470,8 @@ def set_dating_status(self, status=None):
 
         self.item = self.dynamo.set_user_dating_status(self.id, status)
         return self
+
+    def update_last_found_users_at(self, now=None):
+        now = now or pendulum.now('utc')
+        self.dynamo.set_user_last_found_users_at(self.id, now)
```

Comment:
```suggestion
        self.dynamo.set_user_last_found_users_at(self.id, now=now)
```

For the same reason as the previous comment.